apps/explorer/tests/transaction.spec.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { expect, test } from '@playwright/test';
+
+import { faucet, mint } from './utils/localnet';
+
+test('displays the transaction timestamp', async ({ page }) => {
+    const address = await faucet();
+    const tx = await mint(address);
+    const txid = tx.EffectsCert.certificate.transactionDigest;
+    await page.goto(`/transaction/${txid}`);
+    await expect(
+        page.getByTestId('transaction-timestamp').locator('div').nth(1)
```

Comment:
Can we just use `not.toBeEmpty()` at the root instead of locating into the div? I don't love depending on DOM structure (not a blocking concern, we have other tests that do this, just ideally want to minimize it).